cms/admin/pageadmin.py

```diff
@@ -1194,8 +1194,13 @@ def change_template(self, request, object_id):
 
         to_template = request.POST.get("template", None)
```

Comment:
What does "to" mean in this context? I'm trying to make sense of the variable naming.

Is it something like the page content is going to go in to this template?

Reply:
@marksweb Good question. I assume it should signify the value that goes into the `template` of the page content object. This, of course, is the template that the page content will be rendered in. 

A more traditional way of naming would just be `template`.